src/main/java/com/vesta/service/converter/UserConverter.java

```diff
@@ -0,0 +1,19 @@
+package com.vesta.service.converter;
+
+import com.vesta.repository.entity.UserEntity;
+import com.vesta.service.dto.UserDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserConverter {
+
+    public UserDto converter(UserEntity entity) {
+        UserDto dto = new UserDto();
```

Comment:
entity can be null so add if(entity == null) return null; 
the same for the second converter